controllers/component_build_controller.go

```diff
@@ -173,9 +175,20 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		log.Info("Pipelines as Code enabled")
 
 		// Obtain Pipelines as Code callback URL
-		webhookTargetUrl, err := r.getPaCRoutePublicUrl(ctx)
-		if err != nil {
-			return ctrl.Result{}, err
+		webhookTargetUrl := os.Getenv(pipelinesAsCodeRouteEnvVar)
+		if webhookTargetUrl == "" {
+			// The env variable is not set
+			if req.ClusterName != "" {
```

Comment:
```suggestion
			if req.ClusterName == "" {
```

if the string is empty, then it is not set, right?

Reply:
The comment is related to the if above: `if webhookTargetUrl == "" ` that happens after an attempt to read the env variable.

Reply:
I'll add an empty line to separate

Reply:
ah ok I see making this resolved